nano/templates/crew_monitor.tmpl

```diff
@@ -72,7 +72,7 @@ Used In File(s): code\modules\modular_computers\file_system\programs\medical\sui
 				<td><span class="white">Poisoning:</span> Not Available</td>
 			{{/if}}
 			{{if value.sensor_type >= 3}}
-				<td><span class="white">Coordinates:</span> {{:value.x}}, {{:value.y}}, {{:value.z}}</td>
+				<td><span class="white">Coordinates:</span> {{:value.x / 1.285}}, {{:value.y / 1.285}}, {{:value.z}}</td>
```

Comment:
These values should be passed as data from the define in `code/datums/repositories/crew/tracking.dm`